lumicks/pylake/tests/test_image.py

```diff
@@ -40,11 +40,11 @@ def test_timestamps_image(num_lines, pixels_per_line, pad_size):
     pad = np.zeros(pad_size, dtype=np.int32)
     infowave = np.hstack([np.hstack([pad, line_info_wave, pad]) for _ in np.arange(num_lines)])
     start_indices = np.hstack([np.hstack([pad, line_selector, pad]) for _ in np.arange(num_lines)])
-    time = np.arange(len(infowave), dtype=np.int64) + 100
+    time = np.arange(len(infowave), dtype=np.int64) * int(700e9) + 1623965975045144000
```

Comment:
I feel you need a comment to explain why you chose to initialize `time` like that, and what kind of scenario this initialization is testing. Are you effectively testing `np.arrange()` here?

Reply:
No, it's making sure the timestamps are in a large enough range that roundoff errors larger than 1 ns start happening from a conversion to floating point. I'll add a comment:

```
    # It is important to use timestamps that are large enough such that floating point
    # round-off errors will occur if the data is converted to floating point representation.
```